### PR TITLE
Better forms and email sending

### PR DESCRIPTION
**Pull request by noplanman**
Improve HTML form output.
Better email message sending.
Various code fixes and simplifications.

@dmorley 
~**Don't merge yet**, need to test this one properly first.
Thought I'd open the PR so that you can already have a look and comment on it.~

### CODE REVIEW

File: db/edit.php [hunk 1]
```diff
@@ -23,55 +21,56 @@
...
-    $headers = "From: " . $adminemail . "\r\nCc:" . $adminemail . "," . $_oldemail . "\r\n";
-    @mail($to, $subject, $message, $headers);
+    $result || die('Error in SQL query: ' . pg_last_error());
     pg_free_result($result);
     pg_close($dbh);
+
+    $to      = $_email;
+    $subject = 'Edit notice from poduptime';
+    $message = 'Data for ' . $_domain . ' Updated. If it was not you reply and let me know!';
+    $headers = ['From: ' . $adminemail, 'Cc: ' . $row['oldemail'], 'Bcc: ' . $adminemail];
```
Comment (dmorley) [inline db/edit.php]: Not clicking for me how this will have a value $row['oldemail']

Comment (noplanman) [inline db/edit.php]: From `while ($row = pg_fetch_array($result)) {` on line 23

Comment (dmorley) [inline db/edit.php]: :question: edit.php for me shows: $sql    = 'SELECT domain,email,token,tokenexpire,stats_apikey,weight,terms FROM pods WHERE domain = $1';
So you must have modified that but I don't see anything here in the changes where you added oldemail

Comment (noplanman) [inline db/edit.php]: D'oh! Of course that should be `$row['email']`, whoops.

**Comment by noplanman** [comment]
@dmorley Ok, this should be ready to go, please check it out.

**Comment by dmorley** [comment]
Are the conflicts now from your other PRs? They are not in develop so develop is conflicting with this branch?

**Comment by noplanman** [comment]
Oh, will do a rebase, just a sec.

**Comment by noplanman** [comment]
Should be ok now.

**Pull request merged**